mlrun/config.py

```diff
@@ -269,6 +269,17 @@
             # - mlrun.runtimes.function.enrich_function_with_ingress
             "add_templated_ingress_host_mode": "never",
         },
+        "logs": {
+            "pipelines": {
+                "pull_state": {
+                    "mode": "disabled",
+                    # those params are used when mode is enabled
+                    "pull_logs_interval": 30,  # seconds
+                    "pull_state_interval": 5,  # seconds
+                },
+            },
+            "pull_logs_default_interval": 3,  # seconds
+        },
```

Comment:
now these values are used, it would be helpful to add some comments explaining their difference